README.md

```diff
@@ -88,6 +86,42 @@ You will find a **server-rendering** example in the repo´s example directory.
 
 redux-router will notice if the router state in your Redux store changes from an external source other than the router itself — e.g. the Redux Devtools — and trigger a transition accordingly!
 
+## Differences with React-Router-Redux
+
+#### React-Router-Redux
+
+[react-router-redux](https://github.com/reactjs/react-router-redux) (formerly redux-simple-router) takes a different approach to
+integrating routing with redux. react-router-redux lets React-Router do all the heavy lifting and syncs the url data to a history
+[location](https://github.com/reactjs/history/blob/master/docs/Location.md#location) object in the store. This means that users can use
+React-Router's APIs directly and benefit from the wide array of documentation and examples there.
+
+The README for react-router-redux has a useful picture included here:
+
+redux (store.routing)  ↔  react-router-redux  ↔  history (history.location)  ↔  react-router
```

Comment:
I suggest copying in the HTML entities. Makes working with the Markdown a little less problematic:
```md
[redux](https://github.com/rackt/redux) (`store.routing`) &nbsp;&harr;&nbsp; [**react-router-redux**](https://github.com/reactjs/react-router-redux) &nbsp;&harr;&nbsp; [history](https://github.com/reactjs/history) (`history.location`) &nbsp;&harr;&nbsp; [react-router](https://github.com/reactjs/react-router)
```